.github/workflows/test.yml

```diff
@@ -159,7 +159,7 @@ jobs:
         uses: cypress-io/github-action@v6
         with:
           working-directory: frontend
-          config-file: CI/ESS/e2e/cypress.nestjs.json
+          config-file: CI/ESS/e2e/cypress.github.ts
```

Comment:
I have a question.
Can we remove the cypress.nest.json from FE after this change?
It looks like currently we have 3 cypress.xxx.ts file with almost exact configuration values

Reply:
No, this uses the one from frontend (check the working directory)

But I agree, we should create a single e2e setup.

Reply:
I created #1187